src/qt/guiutil.cpp

```diff
@@ -749,6 +749,19 @@ QString boostPathToQString(const fs::path &path)
     return QString::fromStdString(path.string());
 }
 
+QString ConnectionTypeToQString(ConnectionType conn_type)
+{
+    switch (conn_type) {
+    case ConnectionType::INBOUND: return "Inbound full relay";
```

Comment:
571b0469fa952be98874d1afd772a7fa708dd83f

It seems not always true :)

IIUC, you recent changes in `bitcoin-cli` distinguish "block relay" for inbound connections as well.

Reply:
> IIUC, you recent changes in `bitcoin-cli` distinguish "block relay" for inbound connections as well.

Yes, I'm anticipating `inbound full relay` and `inbound block relay` connection types (https://github.com/bitcoin/bitcoin/pull/20726 and https://github.com/bitcoin/bitcoin/pull/20729) but I guess it's better to revert to just inbound for now and update it later.

Reply:
(in the latest -netinfo update in https://github.com/bitcoin/bitcoin/pull/20764, the issue is avoided by using the connection type only to parse full/block/manual/feeler/addr, but not for the direction)

Reply:
In the latest -netinfo update in https://github.com/bitcoin/bitcoin/pull/20764, the issue is avoided by using the connection type only to parse full/block/manual/feeler/addr, but not for the direction. Maybe best to do the same here, hm.

Reply:
Now "Inbound"